enthought_example/example_plugin.py

```diff
@@ -58,5 +58,6 @@ def get_data_views(self):
         # This functionality is optional (the plugin can be run on the
         # bdss without a GUI), so this quite expensive import is done
         # inside the method.
-        from .example_data_views import ExampleCustomPlot
+        from enthought_example.example_data_views.example_data_view import \
```

Comment:
This seems fine, it may not look as neat but it clearly shows what is being imported when `force_wfmanager` is required